router.go

```
@@ -375,6 +387,56 @@ func (app *App) register(methods []string, pathRaw string, group *Group, handler
+}
+
+// RemoveRoute is used to remove a route from the stack
+// TODO should Remove Route only be called dynamically? Can you use it to remove the /define route, leaving the app without
+// any thing on the stack but the GET /hello Route? prefix /he?
+func (app *App) RemoveRoute(path string, methods ...string) {
+	app.mutex.Lock()
+	defer app.mutex.Unlock()
+	app.removeRoute(path, methods)
+}
+
+func (app *App) removeRoute(path string, methods []string) {
```

golangci-lint (1.62.2)

[warning] confusing-naming: Method 'removeRoute' differs only by capitalization to method 'RemoveRoute' in the same source file

(revive)